books/free-programming-books-es.md

```diff
@@ -110,6 +110,7 @@
 * [Docker](https://github.com/brunocascio/docker-espanol#docker) (Github)
 * [El camino a un mejor programador](http://emanchado.github.io/camino-mejor-programador/downloads/camino_2013-01-19_0688b6e.html) (HTML) [(PDF, ePub)](http://emanchado.github.io/camino-mejor-programador/)
 * [Introducción a Docker](https://www.rediris.es/tecniris/archie/doc//TECNIRIS47-1b.pdf) (PDF)
+* [Los Apuntes de Majo](https://losapuntesdemajo.vercel.app) - Majo Ledesma (HTML)
```

Comment:
Hey @majoledesma ya entraste en nuestra lista.

:information_source: Links inside points to Google Drive

Reply:
Acabo de ver que en la guía de contribución no se admiten recursos en Google Drive, en ese caso este PR se cerraría por ahora ¿correcto? 🤔

Reply:
En este caso creo que GDrive sería válido ya que es como Majo lo mantiene

Reply:
Si puedes responder a la pregunta de Eric estaría genial